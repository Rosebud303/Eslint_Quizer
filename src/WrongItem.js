import React from 'react'

export default function WrongItem({name, answer, id}) {
  return (
    <div key={answer}>
      <h2>{name}</h2>
      <p>This is the correct way to perform question :{id}</p>
      <img src={answer}/>
    </div>
  )
}
