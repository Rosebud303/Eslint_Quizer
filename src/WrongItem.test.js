import React from 'react';
import WrongItem from './WrongItem.js';
import { shallow } from 'enzyme';

const mockData = {
    name: 'Bob',
    answer: 'Some Image',
    id: 1
}

describe('WrongItem', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <WrongItem {...mockData}/>
        )
    });

    it('Should match the snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    });
});