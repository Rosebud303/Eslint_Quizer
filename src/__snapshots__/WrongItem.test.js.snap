// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WrongItem Should match the snapshot 1`] = `
<div
  key="Some Image"
>
  <h2>
    Bob
  </h2>
  <p>
    This is the correct way to perform question :
    1
  </p>
  <img
    src="Some Image"
  />
</div>
`;
